sympy/utilities/lambdify.py

```diff
@@ -50,9 +50,9 @@
     "E": "e",
     "I": "j",
     "ln": "log",
-    #"lowergamma":"lower_gamma",
+    "lowergamma":"gammainc",
```

Comment:
Those arguments are set by the mpmath printers, and the current tests are running the printers only. They do not make use of the contents of MPMATH_TRANSLATIONS.

If the translations are added to MPMATH_TRANSLATIONS, it should be tested that the same translation `gammainc` really works for both functions and does not always produce the same values for `uppergamma` and `lowergamma`.